src/main/java/aztech/modern_industrialization/compat/kubejs/material/MIMaterialKubeJSEvents.java

```diff
@@ -32,4 +32,5 @@ public interface MIMaterialKubeJSEvents {
 
     EventHandler ADD_MATERIALS = EVENT_GROUP.startup("addMaterials", () -> AddMaterialsEventJS.class);
     EventHandler MODIFY_MATERIAL = EVENT_GROUP.startup("modifyMaterial", () -> ModifyMaterialEventJS.class).extra(Extra.STRING);
+    EventHandler ADD_CABLE_TIERS = EVENT_GROUP.startup("cableTiers", () -> AddCableTiersEventJS.class);
```

Comment:
Wasn't too sure where to put this; went with materials (as its mostly related to that). Definitely open to changing it if anyone has a better idea.

Reply:
I moved it to registration events